### PR TITLE
Day 19 august

### PR DESCRIPTION
**Pull request by Basma-Abdallah**

### CODE REVIEW

**Comment by Nora-Lootfy** [comment]
wow, lots of conflicts😂

have an idea how to resolve it locally or in github?

**Comment by Nora-Lootfy** [comment]
this is something in common when working as a group

File: package.json [hunk 1]
```diff
@@ -6,8 +6,11 @@
         "build": "vite build"
     },
     "devDependencies": {
+        "@popperjs/core": "^2.11.6",
```
Comment (Nora-Lootfy) [inline package.json:9]: someone is working with npm😂

File: resources/views/contact.blade.php
Comment (Nora-Lootfy) [inline resources/views/contact.blade.php:1]: personally, i don't like these extra empty lines, it makes code less readable

File: routes/web.php [hunk 2]
```diff
@@ -1,7 +1,10 @@
 <?php
 
 use App\Http\Controllers\PublicController;
+use Illuminate\Http\RedirectResponse;
```
Comment (Nora-Lootfy) [inline routes/web.php:4]: also it don't prefer adding use of classes without the need for it, it gives me an idea that you are using this class so i scroll down to see where you use it 

if you find this wrong just forget about it, you don't have to follow my conventions

**Review by Nora-Lootfy — COMMENTED** [review]
I will leave just a comment as all my review is about recommendations, you don't have to follow it

All looking good, thanks for hard working ya @Basma-Abdallah

File: resources/views/contact.blade.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (Omneya-Abdelmajeed) [inline resources/views/contact.blade.php:1]: Basma please add this:
@push('pageHeader')
Contact
@endpush
here at the top of this file as the header shows 'Contact' not 'About us' as Nora recommended before

File: resources/views/contact.blade.php
Comment (Omneya-Abdelmajeed) [inline resources/views/contact.blade.php:1]: Basma please add this:
@Push('pageHeader')
Contact
@endpush
here at the top of this file as the header shows 'Contact' not 'About us' as Nora recommended before

**Pull request merged**